packages/composite-checkout/src/components/checkout-steps.tsx

```diff
@@ -506,6 +526,10 @@ export function CheckoutStepArea( {
 					disabled={ isThereAnotherNumberedStep || disableSubmitButton }
 					onLoadError={ onSubmitButtonLoadError }
 				/>
+				<SubmitButtonFooter>
+					<img src={ badge14Src } alt="" />
+					<span>{ __( '14 day money back guarantee' ) }</span>
```

Comment:
I don't think that this guarantee applies to all possible uses of the `composite-checkout` module. Although wpcom is currently the  only user of this package, it's designed with no knowledge of wpcom and is intended to be used with other services as well. I think this might do better if we put this badge in calypso.

Reply:
Thanks @sirbrillig. I'll clarify which products are impacted by this guarantee.